examples/workflow/aio/context_interceptors_example.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+"""
+Example: Interceptors for context propagation (client + runtime).
+
+This example shows how to:
+ - Define a small context (dict) carried via contextvars
+ - Implement ClientInterceptor to inject that context into outbound inputs
+ - Implement RuntimeInterceptor to restore the context before user code runs
+ - Wire interceptors into WorkflowRuntime and DaprWorkflowClient
+
+Note: Scheduling/running requires a Dapr sidecar. This file focuses on the wiring pattern.
```

Comment:
can you add a blurb here on why an end user would want to do this interceptor pls? Like for propagating trace or X, Y, Z. We also need to update docs somewhere to share that this is a new feature. Is there a doc in the python sdk somewhere for this or do we just add in dapr docs somehwere?

Reply:
not in python docs, but the README should have references to it.